docs/building-rpms-from-source.md

```diff
@@ -0,0 +1,166 @@
+# Building RPMs from source with mkosi
+
+If you want to build and RPM from source and install it within a mkosi
```

Comment:
```suggestion
If you want to build an RPM from source and install it within a mkosi
```

Reply:
Incidentally, I think it should also be *an* mkosi, because the m in the acronym starts with a vowel sound.
```suggestion
If you want to build and RPM from source and install it within an mkosi
```

Reply:
Hmm, I'm not sure on this one, "a mkosi image" sounds much more natural to me than "an mkosi image"

Reply:
To me it's the other way 'round, "an [e]m" flows more easily than "a [e]m", since the "m" beings with a vowel sound, that's why I noted it.

Reply:
Yeah, "an mkosi thingy" would be the usual, unless you pronounce mkosi staring with a consonant, which would be hard, since there's another consonant right after.